lightcurve_fitting/speccal.py

```diff
@@ -464,7 +464,8 @@ def create_wiserep_tsv(specpaths, wiserep_dir, verbose=False):
     # collect the metadata
     bibcode = input('bibcode: ')
     rows = []
-    instruments = {}
+    if not instruments:
```

Comment:
`if instruments is None` would be a tiny bit safer I guess

Reply:
fixed!